doc/source/reference/frame.rst

```diff
@@ -16,17 +16,11 @@ Constructor
 
 Attributes and underlying data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-**Axes**
```

Comment:
If this was part of trying to see if a source link could be added, could you revert the changes in this file?

Reply:
Done